include/nbl/asset/IMeshPacker.h

```diff
@@ -78,8 +32,23 @@ class MeshPackerBase
     };
 
 protected:
-    MeshPackerBase(const AllocationParams& allocParams)
-        :m_allocParams(allocParams) {};
+    MeshPackerBase(uint16_t minTriangleCountPerMDIData, uint16_t maxTriangleCountPerMDIData)
+        :m_maxTriangleCountPerMDIData(maxTriangleCountPerMDIData),
+         m_minTriangleCountPerMDIData(minTriangleCountPerMDIData),
+         m_MDIDataAlctrResSpc(nullptr),
+         m_idxBuffAlctrResSpc(nullptr),
+         m_vtxBuffAlctrResSpc(nullptr)
+    {
+        assert(minTriangleCountPerMDIData <= 21845);
+        assert(maxTriangleCountPerMDIData <= 21845);
+    };
+
+    virtual ~MeshPackerBase()
+    {
+        _NBL_ALIGNED_FREE(m_MDIDataAlctrResSpc);
+        _NBL_ALIGNED_FREE(m_idxBuffAlctrResSpc);
+        _NBL_ALIGNED_FREE(m_vtxBuffAlctrResSpc);
+    }
 
     static bool cmpVtxInputParams(const SVertexInputParams& lhs, const SVertexInputParams& rhs)
```

Comment:
V2 doesn't need this, move from Base to V1

P.S. is this function even used anymore?

Reply:
yes
https://github.com/Przemog1/Nabla/blob/mesh_packer/include/nbl/asset/IMeshPacker.h#L167